2019-12-27-what-is-http-messages.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Learning http messages"
+date: "2019-12-27 11:40:00 UTC"
+daft: true
+tags:
+  - http messages
+  - http requests
+  - http responses
+  - http
+---
+
+I worked as a frontend web dev for the past 2 years, that means that I'm only playing around with the data that was already provided to me.
+2 months ago, my friend has offered me a contract at his company as a backend dev, and my first task was working on an <a href="https://github.com/curveball/a12n-server">authentication server</a> for the open-source project <a href="https://github.com/curveball">framework</a> the team was building. When I was a frontend dev,these data was just there for me, "magically", waiting for me to use them, but now I have to build them from scratch.
```

Comment:
This is a small comment, but you _could_ use markdown links instead of in-line links. Both is fine, it's a preference thing

Reply:
The sentence is also a bit weird, look at it in a browser